.changeset/sour-bats-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store": minor
```

Comment:
```suggestion
"@latticexyz/store": patch
```

We tend to not follow semver _exactly_ and sneak new features into patch changes rather than minor and save minor bumps for big changes.